samples/todolist_failure_restore.md

```diff
@@ -0,0 +1,165 @@
+# Pelorus Demo
+
+## Mean Time to Restore and Change Failure Rate
+
+* "Mean Time to Restore" utilizes the failure exporter to measure bug resolution and bug resolution time stamp.
+* "Change Failure Rate" utilizes the failure exporter to measure bug creation and creation time stamp.
+
+## Todolist application
+* We'll use the same forked Github repository that was used previously in the todolist-mongo example.  There is no need to refork the konveyor/mig-demo-apps git repository if you have a fork.
+  *  Fork [the konveyor mig-demo-apps repo](https://github.com/konveyor/mig-demo-apps) to https://github.com/your_org/mig-demo-apps
+  * Ensure that `Github Issues` are enabled in your Github settings.
+     * https://github.com/your_org/mig-demo-apps/settings
+
+## Additional Reference
+* [The definitive Pelorus guide can be found on our readthedocs page.](https://pelorus.readthedocs.io/)
+
+## Assumptions
+- oc command line tools installed
+- Logged into OCP Cluster
+- Pelorus is installed and running
+- The [todolist-mongo](todolist_commit_deploy.md) sample is deployed and the Lead Time for Change and Deployment Frequency measurements are monitoring the todolist application.
+- Github issues are enabled in https://github.com/your_org/mig-demo-apps/settings
+
+![start](images/todolist_failure_restore_start.png)
+
+## Goals
+- Create and resolve bugs in Github issues that exercise Pelorus Metrics
+
+
+## Pelorus configuration
+1. Make a copy of [the values.yaml file in the pelorus repo](https://github.com/konveyor/pelorus/blob/master/charts/pelorus/values.yaml) ([raw link for curl-ing](https://raw.githubusercontent.com/konveyor/pelorus/master/charts/pelorus/values.yaml)) and save it to /var/tmp/values.yaml
+```
+cp charts/pelorus/values.yaml /var/tmp/
+```
+2. Update the /var/tmp/values.yaml to match the following text. You may already have the deploytime and committime exporters configured from a previous example.
+
+>Note: A users [Github personal access token](https://github.com/settings/tokens) is required
+>Note: The `PROJECTS` key's value is the fork of the mig-apps-demo repository.
+
+
+```yaml
+exporters:
+  instances:
+  - app_name: deploytime-exporter
+    source_context_dir: exporters/
```

Comment:
With this change we really don't need to have source_context_dir nor source_url. Also LOG_LEVEL debug is not something we want in standard deployment . I would also like to ensure people will start using exporter_type, which is already documented here: https://github.com/konveyor/pelorus/blob/master/docs/Configuration.md#configuring-exporters

https://github.com/konveyor/pelorus/commit/ca188e5a8aa7114467a193ea022d91363b007b24

Having said above, the file should look like:
```yaml
exporters:
  instances:
  - app_name: deploytime-exporter
    exporter_type: deploytime
    source_ref: master
    extraEnv:
    - name: NAMESPACES
      value: mongo-persistent
  - app_name: committime-exporter
    exporter_type: committime
    source_ref: master
  - app_name: failure-exporter
    exporter_type: failure
    source_ref: master
    extraEnv:
     - name: PROVIDER
       value: github
     - name: TOKEN
       value: ghp_J<snip>
     - name: PROJECTS
       value: <your_org>/mig-demo-apps
```

Reply:
ah.. thanks for taking the time to paste an update!  fixing now